tests/data/apply_filters.php

```diff
@@ -105,102 +107,63 @@
  *
  * @param float|int|null $foo Hello, World.
  */
-$value = (int) apply_filters('filter',$foo);
+$value = (int)apply_filters('filter', $foo);
 assertType('int', $value);
 
 /**
  * Global class that's been imported.
  *
- * @param WP_Post|null $foo Hello, World.
+ * @param \WP_Post|null $foo Hello, World.
  */
-$value = apply_filters('filter',$foo);
+$value = apply_filters('filter', $foo);
 assertType('WP_Post|null', $value);
 
 /**
  * Global class that's not been imported.
  *
  * @param \WP_Term|null $foo Hello, World.
  */
-$value = apply_filters('filter',$foo);
+$value = apply_filters('filter', $foo);
 assertType('WP_Term|null', $value);
 
 /**
  * Constant string type.
  *
  * @param 'aaa'|'bbb' $foo Hello, World.
  */
-$value = apply_filters('filter',$foo);
+$value = apply_filters('filter', $foo);
 assertType("'aaa'|'bbb'", $value);
 
 /**
  * Typed array passed through `list()`.
  *
  * @since 2.5.0
  *
- * @param int[]        $max_image_size {
- *     An array of width and height values.
- *
- *     @type int $0 The maximum width in pixels.
- *     @type int $1 The maximum height in pixels.
+ * @param array<int> $max_image_size {
+ * An array of width and height values.
+ * @type int $0 The maximum width in pixels.
+ * @type int $1 The maximum height in pixels.
```

Comment:
This shouldn't change because this test is to explicitly check this format of docblock. I think it makes more sense to ignore this file and its included files because it's likely to include code that intentionally does not adhere to the coding standards.

Reply:
> this format

Meaning WP core's own invention.

Reply:
And the `array{ ... }` syntax used by PHPStan and Psalm is somebody else's invention, it's not part of FIG or phpDoc.

Reply:
Regardless, it's irrelevant. This extension is for supporting syntax used in the WordPress ecosystem so its tests should reflect that.

Reply:
sure, makes sense to me. I missed that one after the auto-fix. adapted